lang/fr.json

```diff
@@ -390,7 +390,11 @@
     "defaultMessage": "Less",
     "description": "Less button text (showing less content)"
   },
-  "load-error-makecode": {
+  "load-error-makecode-info": {
+    "defaultMessage": "This hex file cannot be loaded in the Python Editor. The Python Editor cannot open hex files created with Microsoft MakeCode.",
+    "description": "Load error message"
+  },
+  "load-error-makecode-link": {
     "defaultMessage": "This hex file cannot be loaded in the Python Editor. The Python Editor cannot open hex files created with Microsoft MakeCode.",
     "description": "Load error message"
   },
```

Comment:
This will look a bit daft for French.

Reply:
Oops, will fix.

Reply:
bdeddd1138db97dd701a3f6b739d32a4cd6ec365